assets/app/view/game/buy_trains.rb

```diff
@@ -92,9 +97,16 @@ def render
           },
         }
 
-        if (step.can_buy_train?(@corporation) && step.room?(@corporation)) || must_buy_train
-          children << h(:div, "#{@corporation.name} must buy an available train") if must_buy_train
-          if should_buy_train == :liquidation
+        if @corporation.system?
+          children << render_shells
+        else
+          store(:active_shell, nil)
```

Comment:
do you want this to cause a rerender?

Reply:
To be honest, I don't understand what's going on under-the-hood with settings. How long is the setting around for? If an action occurs, is it wiped? That's what I'm trying to do here.

Reply:
add skip: true

Reply:
you need to add skip true